.github/workflows/docker-build.yaml

```diff
@@ -11,91 +11,86 @@ on:
+            DOCKER_ELASTICSEARCH_IMAGE_EXISTS: ${{ steps.variables.outputs.DOCKER_ELASTICSEARCH_IMAGE_EXISTS }}
+            BRANCH_NAME: ${{ steps.variables.outputs.BRANCH_NAME }}
+            BRANCH_NAME_ESCAPED: ${{ steps.variables.outputs.BRANCH_NAME_ESCAPED }}
         steps:
             -   name: GIT checkout branch - ${{ github.ref }}
                 uses: actions/checkout@v3
                 with:
                     ref: ${{ github.ref }}
-            -   name: Login to GitHub Container Registry
-                uses: docker/login-action@v2
-                with:
-                    registry: ghcr.io
-                    username: ${{ github.actor }}
-                    password: ${{ secrets.GITHUB_TOKEN }}
-            -   name: Extract branch name to ENV variable BRANCH_NAME
-                run: echo "BRANCH_NAME=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_ENV
-            -   name: Check if PHP-FPM image and Storefront image exists
+            -   name: Extract variables to output
+                id: variables
                 env:
                     DOCKER_USERNAME: ${{ github.repository_owner }}
                     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
                 run: |
                     DOCKER_PHP_FPM_IMAGE_TAG=github-action-`find project-base/app/docker/php-fpm -type f -exec stat -c "%a %n" {} \; -exec cat {} \; | md5sum | awk '{ print $1 }'`
-                    DOCKER_STOREFRONT_IMAGE_TAG=github-action-storefront-${{ github.sha }}
-                    echo "DOCKER_PHP_FPM_IMAGE_TAG=${DOCKER_PHP_FPM_IMAGE_TAG}" >> $GITHUB_ENV
-                    echo "DOCKER_STOREFRONT_IMAGE_TAG=${DOCKER_STOREFRONT_IMAGE_TAG}" >> $GITHUB_ENV
-                    echo "DOCKER_USERNAME=${DOCKER_USERNAME}" >> $GITHUB_ENV
+                    DOCKER_PHP_FPM_REPOSITORY_TAG=ghcr.io/${DOCKER_USERNAME}/php-fpm:${DOCKER_PHP_FPM_IMAGE_TAG}
                     DOCKER_PHP_FPM_IMAGE_EXISTS=`. .github/check-image-exists.sh && checkImageExists ${DOCKER_USERNAME} php-fpm ${DOCKER_PHP_FPM_IMAGE_TAG} ${GITHUB_TOKEN} > /dev/null ; echo $?`
-                    echo "DOCKER_PHP_FPM_IMAGE_EXISTS=${DOCKER_PHP_FPM_IMAGE_EXISTS}" >> $GITHUB_ENV
-            -   name: Build PHP-FPM image and push it to Github Container Registry
-                if: env.DOCKER_PHP_FPM_IMAGE_EXISTS == 0
-                run: .github/build-php-fpm-image.sh
-            -   name: Build Storefront image and push it to Docker Hub
-                run: .github/build-storefront-image.sh
-            -   name: Check if Elasticsearch image exists
-                env:
-                    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                run: |
+                    DOCKER_STOREFRONT_IMAGE_TAG=github-action-`find project-base/storefront/docker -type f -exec stat -c "%a %n" {} \; -exec cat {} \; | md5sum | awk '{ print $1 }'`-${{ github.sha }}
```

Comment:
I have introduced it because there were some problems, I also have no power over what will change in the Dockerfile in the future and prefer to use it as it is. Also in the next PR I am creating it without USER, co it correctly works with GitHub actions.

I have removed md5 computing